src/AppBundle/Controller/BlogController.php

```diff
@@ -56,6 +56,16 @@ public function indexAction($page)
      */
     public function postShowAction(Post $post)
     {
+        // Symfony provides a function called 'dump()' which is an improved version
+        // of the well-known 'var_dump()' function. It's useful to show the contents
+        // of any variable and check if they contain what you expect.
+        // The 'dump()' function is not available in the 'prod' environment, so use
+        // it only while developing the application, not in production. To make this
+        // app work in both environments, we check the value of the 'kernel.debug' parameter.
+        if (true === $this->getParameter('kernel.debug')) {
```

Comment:
@voronkovich in a real application, that trick is nice. However, in this app it may be confusing:

1) If users see the trick, they might think that they need to do that in their apps.
2) If they don't see the trick, they might think that `dump()` always works but has a random behavior (the output change without reason).